.github/requirements.txt

```diff
@@ -0,0 +1,6 @@
+python-frontmatter>=0.4.2
+feedparser>=5.2.1
+pyaml>=17.12.1
+# sgmllib not in python 3; workaround solution for feedparser relative link resolution
+sgmllib3k
+beautifulsoup4
```

Comment:
You shouldn’t need to update this file - the versions are the same and bs4 I added for some extended parsing (not used here!)

Reply:
👍 no soup 🥣